.github/workflows/docker-build.yaml

```diff
@@ -789,12 +789,15 @@ jobs:
             STANDARDS_STOREFRONT_RESULT: ${{ needs.standards-storefront.result }}
             TRANSLATIONS_DUMP_RESULT: ${{ needs.translations-dump-check.result }}
             REVIEW_RESULT: ${{ needs.review.result }}
+            CHECK_CONSOLE_COMMANDS_RESULT: ${{ needs.check-console-commands.result }}
+            TESTS_STOREFRONT_ACCEPTANCE_RESULT: ${{ needs.tests-storefront-acceptance.result }}
+            UNIT_TESTS_STOREFRONT_RESULT: ${{ needs.unit-test-storefront.result }}
```

Comment:
should not these three jobs be added to the `needs` section?

Reply:
You are correct! I have added them and squashed it.